tock/tock/settings/dev.py

```diff
@@ -1,28 +1,49 @@
-from .base import *  # noqa
-
 import dj_database_url
 
+from .base import *  # noqa
+
 DEBUG = True
-TEMPLATE_DEBUG = True
+TEMPLATE_DEBUG = DEBUG
+DEBUG_TOOLBAR_PATCH_SETTINGS = False
 
-INTERNAL_IPS = ('127.0.0.1',)
+INTERNAL_IPS = [
+    '127.0.0.1',
+    '::1',
+]
```

Comment:
This was partly already there and partly due to the instructions in [Django Debug Toolbar's setup](https://django-debug-toolbar.readthedocs.io/en/1.4/installation.html#internal-ips).  Due to the nature of how this has to be run because of Docker though, we may not need to set these.

Yes, `::1` is the IPv6 version of [localhost](https://en.wikipedia.org/wiki/Localhost#Name_resolution).